src/client/components/evaluate/ProjectEvaluateSidebar.tsx

```diff
@@ -80,12 +80,26 @@ const ProjectEvaluateSidebar = ({
     {
       key: "equalPopulation",
       name: "Equal Population",
-      status: false,
+      status:
+        (avgPopulation &&
+          geojson &&
+          geojson?.features.filter(f => {
+            return (
+              f.id !== 0 &&
+              // @ts-ignore
+              f.properties.percentDeviation &&
+              // @ts-ignore
+              Math.abs(f.properties.percentDeviation) <= popThreshold
```

Comment:
There aren't any type errors here?
```suggestion
              f.properties.percentDeviation &&
              Math.abs(f.properties.percentDeviation) <= popThreshold
```

Reply:
Realizing now you copy/pasted this from below where the `@ts-ignore` comments are... also not needed?